### DISCUSSION TITLE
Beta release 0.1.0b3

### DISCUSSION THREAD

**Discussion by haipinglu** [pykale/pykale#103] [General] [answered]
#### Code Improvements

* [#84](https://github.com/pykale/pykale/pull/84): Auto assign to the default project
* [#91](https://github.com/pykale/pykale/pull/91): MPCA pipeline
* [#93](https://github.com/pykale/pykale/pull/93): Fix black config and rerun
* [#97](https://github.com/pykale/pykale/pull/97): Add changelog CI and update logo

#### Dependencies

* [#82](https://github.com/pykale/pykale/pull/82): Remove requirements in examples and update setup

#### Tests

* [#70](https://github.com/pykale/pykale/pull/70): Add tests for utils.print
* [#80](https://github.com/pykale/pykale/pull/80): Extend automated test matrix and rename lint
* [#85](https://github.com/pykale/pykale/pull/85): Test utils logger
* [#87](https://github.com/pykale/pykale/pull/87): Test cifar/digit_access and downgrade black
* [#90](https://github.com/pykale/pykale/pull/90): Test mpca
* [#94](https://github.com/pykale/pykale/pull/94): Update test guidelines

#### Documentation Updates

* [#81](https://github.com/pykale/pykale/pull/81): Docs update version and installation
* [#88](https://github.com/pykale/pykale/pull/88): Automatically sort documented members by source order
* [#89](https://github.com/pykale/pykale/pull/89): Disable automatic docstring inheritance from parent class

<hr /><em>This discussion was created from the release <a href='https://github.com/pykale/pykale/releases/tag/0.1.0b3'>Beta release 0.1.0b3</a>.</em>

**Comment by bobturneruk** [comment, marked as answer]
Consider updating contributing guidelines to reflect the way in which labels and PR titles are now used to auto-generate changelogs.
